src/content/docs/en/recipes.mdx

```diff
@@ -30,3 +30,4 @@ Add your own here! See our [recipes contributing guide](https://github.com/witha
 - [Generate localized sitemaps for your Astro and Storyblok project](https://straffesites.com/en/blog/localized-sitemap-astro-storyblok)
 - [Integrate Prettier with Astro and Tailwind CSS](https://straffesites.com/en/blog/integrate-prettier-astro-tailwindcss)
 - [Add an RSS feed to Astro using Storyblok's stories](https://straffesites.com/en/blog/rss-feed-astro-storyblok)
+- [Using TRPC in Astro](https://www.thomasledoux.be/blog/using-trpc-astro-islands-react)
```

Comment:
Nit: the official name is "tRPC" according to their official site, I would change this for consistency's sake.
```suggestion
- [Using tRPC in Astro](https://www.thomasledoux.be/blog/using-trpc-astro-islands-react)
```